en/get-started.md

```diff
@@ -317,6 +317,14 @@ Before proceeding, make sure the following requirements are satisfied:
             helm install --namespace tidb-admin --name tidb-operator pingcap/tidb-operator --version v1.1.0
             ```
 
+            For Chinese mainland users, you may use local images:
+
+            ```
+            helm install --namespace tidb-admin --name tidb-operator pingcap/tidb-operator --version v1.1.0 \
+              --set operatorImage=uhub.ucloud.cn/pingcap/tidb-operator:v1.1.0 \
+              --set tidbBackupManagerImage=uhub.ucloud.cn/pingcap/tidb-backup-manager:v1.1.0
```

Comment:
currently, we push tidb-operator images to `uhub.ucloud.cn/pingcap` for every release
once we switch to our new China registry `registry.cn-beijing.aliyuncs.com/tidb` in our CD job, we can update it

Reply:
encountered an issue with `uhub.ucloud.cn/pingcap` the registry